airflow/cli/cli_config.py

```diff
@@ -543,6 +543,15 @@ def string_lower_type(val):
 ARG_DEFAULT = Arg(
     ("-d", "--default"), metavar="VAL", default=None, help="Default value returned if variable does not exist"
 )
+ARG_VAR_DESCRIPTION = Arg(
+    (
+        "-desc",
```

Comment:
single hyphen it is stands for single letter argument (eg: `-o`, `-j`, etc), and better don't use multiple letters.
I'm not sure about argument parser which used in airflow cli, but in general:

```bash
somecommand -desc
```

Might be the same as

```bash
somecommand -d -e -s -c
```

Reply:
Hi @Taragolis ,

Do you recommend if the argument for this should be updated to ignore the `-desc` and only leave `--description` since `-d` is taken by the `default` argument

Reply:
I would recommend not having a short option.

Reply:
Ack, I will deprecate the short argument option and publish a new revision for this

Reply:
This is fixed in the next rev